src/js/app.js

```diff
@@ -423,6 +429,84 @@ $.fn.podlovewebplayer = function webPlayer(options) {
+      'btn_forw_30': '30 seconds forward',
+      'alt_coverart': 'Poster Image',
+      'title_chapter_table': 'Chapter',
+      'title_chapter_table_col_number': 'Chapter number',
+      'title_chapter_table_col_time': 'Start time',
+      'title_chapter_table_col_title': 'Title',
+      'title_chapter_table_col_dur': 'Duration',
+      'tab_title_actions': 'Toggle',
+      'tab_title_message': '$2 $1',
+      'tab_chapter_name': 'Chapter',
+      'tab_downloads_name': 'Downloads',
+      'tab_downloads_btn_download': 'Download',
+      'tab_info_name': 'Info',
+      'tab_info_title_published': 'Published on',
+      'tab_info_title_duration': 'Duration',
+      'tab_info_title_permalink': 'Permalink',
+      'tab_info_title_link_permalink': 'Permalink to this episode',
+      'tab_info_title_link_show': 'Link to show',
+      'tab_info_title_social': 'Keep in touch',
+      'tab_info_text_license': 'The show "$1" ist licensed under',
+      'tab_info_link_license_title': 'See license',
+      'tab_share_name': 'Share',
+      'tab_share_table_caption_share': 'Share podcast',
+      'tab_share_title_what_share': 'What would you like to share?',
+      'tab_share_title_direct_link': 'Direct link',
+      'tab_share_title_share_via': 'Share via ...'
+    },
+    'de': {
+      'link_show': 'Link zur Show',
+      'link_episode': 'Link zur Episode',
+      'btn_play': 'Abspielen',
+      'btn_chapter_prev': 'Zurück zum vorigen Kapitel',
+      'btn_chapter_next': 'Zum nächsten Kapitel springen',
+      'btn_back_30': '30 Sekunden zurück',
+      'btn_forw_30': '30 Sekunden vor',
+      'alt_coverart': 'Poster Image',
+      'title_chapter_table': 'Kapitel',
+      'title_chapter_table_col_number': 'Kapitelnummer',
+      'title_chapter_table_col_time': 'Startzeit',
+      'title_chapter_table_col_title': 'Titel',
+      'title_chapter_table_col_dur': 'Dauer',
+      'tab_title_actions': 'anzeigen / verbergen',
+      'tab_title_message': '$1 $2',
+      'tab_chapter_name': 'Kapitel',
+      'tab_downloads_name': 'Downloads',
+      'tab_downloads_btn_download': 'Download',
+      'tab_info_name': 'Info',
+      'tab_info_title_published': 'Veröffentlicht am',
+      'tab_info_title_duration': 'Dauer',
+      'tab_info_title_permalink': 'Permalink',
+      'tab_info_title_link_permalink': 'Permalink für die Episode',
+      'tab_info_title_link_show': 'Link zur Show',
+      'tab_info_title_social': 'Bleib in Verbindung',
+      'tab_info_text_license': 'Die Show "$1" ist lizensiert unter',
+      'tab_info_link_license_title': 'Lizenz ansehen',
+      'tab_share_name': 'Teilen',
+      'tab_share_table_caption_share': 'Podcast teilen',
+      'tab_share_title_what_share': 'Was möchtest du teilen?',
+      'tab_share_title_direct_link': 'Direkter Link',
+      'tab_share_title_share_via': 'Teilen via ...'
+    }
+  });
+  // $.i18n().locale = 'de'; -> possibility of manually overriding the language, en is fallback
+}
+
```

Comment:
For easier maintenance, I would prefer to release the translations into own & separate data files.